rockcraft.yaml

```diff
@@ -0,0 +1,170 @@
+---
+name: sdcore-bess
+base: ubuntu:20.04
```

Comment:
Is there a reason not to use 22.04?

Reply:
I started with `20.04` to make it work as that is what upstream uses. I have a `wip` locally to upgrade to 22.04, unfortunately it does not build yet.